src/static/css/link.css

```diff
@@ -0,0 +1,37 @@
+a {
+    color: var(--fl-linkColor, var(--link-color));
+    text-decoration: underline;
+}
+
+a:hover {
+    color: var(--fl-linkColor, var(--link-hover-color));
+}
+
+a:active {
+    background: var(--fl-bgColor, var(--link-active-bg));
+    box-shadow: 0 0 0 0.3em var(--fl-bgColor, var(--link-active-bg));
+    color: var(--fl-linkColor, var(--link-active-color));
+}
+
+a:focus {
+    color: var(--fl-linkColor, var(--link-focus-color));
+    outline: 3px solid var(--fl-linkColor, var(--link-focus-color));
```

Comment:
I'd use rem units for these too. We may also want to think about adding them to our scale.